test/unittests/leb128_test.cpp

```diff
@@ -33,6 +33,8 @@ TEST(leb128, decode_u64)
         {{0x81, 0x80, 0x80, 0x80, 0x80, 0x80, 0x80, 0x80, 0x80, 0x00}, 1}, // 1 with max leading zeroes
         {{0xe5, 0x8e, 0x26}, 624485}, // 624485
         {{0xe5, 0x8e, 0xa6, 0x80, 0x80, 0x00}, 624485}, // 624485 with leading zeroes
+        {{0xff, 0xff, 0xff, 0xff, 0x07}, 0x7fffffff},
+        {{0x80, 0x80, 0x80, 0x80, 0x08}, 0x80000000},
```

Comment:
Can you also add 64-bit versions of these?

Reply:
added